base/shell/explorer/taskswnd.cpp

```diff
@@ -1780,7 +1780,7 @@ class CTaskSwitchWnd :
         if (GetForegroundWindow() != TaskItem->hWnd)
             ERR("HandleTaskItemRightClick detected the window did not become foreground\n");
 
-        ::SendMessageCallbackW(TaskItem->hWnd, WM_POPUPSYSTEMMENU, 0, MAKELPARAM(pt.x, pt.y),
+        ::SendMessageCallbackW(TaskItem->hWnd, WM_SYSMENU, 0, MAKELPARAM(pt.x, pt.y),
```

Comment:
Re. This change: I wasn't suggesting you make it. I was just informing about its other alternative name.